pkgs/victoriametrics/victoriametrics/vmutils/scaffold.yaml

```diff
@@ -0,0 +1,10 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/main/json-schema/aqua-generate-registry.json
+# aqua - Declarative CLI Version Manager
+# https://aquaproj.github.io/
+# Other than name is optional. All initial values are just examples.
+name: victoriametrics/victoriametrics/vmutils
+# version_filter: not (Version matches "-rc$")
+# version_prefix: cli-
+all_assets_filter:  (Asset matches "vmutils-") and not (Asset matches "-enterprise") and not (Asset matches "-arm-")
+version_filter: semver("> 1.78.1") and not (Version endsWith "-victorialogs") and (Version endsWith ".0")
```

Comment:
> Version endsWith ".0"

Why is this filter required?

Reply:
It's because the releases other than X.Y.0 only include enterprise assets. For example, [v1.110.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.110.0) has `vmutils-linux-amd64-v1.110.0.tar.gz`, but [v1.110.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.110.1) only includes `vmutils-linux-amd64-v1.110.1-enterprise.tar.gz`.

Without the `endsWith ".0"` filter, `cmdx s` generates a more complex `registry.yaml` and `pkg.yaml`, which seem, at least to me, harder to maintain. However, I'm happy to adjust my PR if you prefer it that way.

Reply:
Oh, I see. Thank you for your explanation.

> the releases other than X.Y.0 only include enterprise assets

Interesting.